content/posts/2023-08-25-pycon-tw-2023-programs.md

```diff
@@ -0,0 +1,59 @@
+Title: PyCon Taiwan 2023 議程總覽
+Date: 2023-08-25 10:00
+Category: programs
+Tags: zh
+Slug: pycon-tw-2023-programs
+Authors: Andy Lee, Ting Yi, Lois, Jaihua
+Summary: ![Programs](images/2023-08-25-pycon-tw-2023-programs/0_programs.jpg){width=400} <br> PyCon Taiwan 2023 將回歸實體年會！趕快來看看今年會有哪些活動及議程！
+
+### 大會議程
+
+請參考 [PyCon TW 2023 議程](https://tw.pycon.org/2023/zh-hant/conference/schedule)。
```

Comment:
Although the main TA of this post is Mandarin users, I think it might still be better to let the browser decide the language. 

```suggestion
請參考 [PyCon TW 2023 議程](https://tw.pycon.org/2023/conference/schedule)。
```